time-to-live.md

```diff
@@ -141,6 +141,8 @@ ALTER TABLE orders TTL_JOB_INTERVAL = '24h';
 
 `TTL_JOB_INTERVAL` is set to `1h` by default.
 
+When executing a TTL job, TiDB will split the table into up to 64 tasks, with the `Region` being the smallest unit. These tasks will be executed distributedly. You can limit the number of TTL tasks that can be executed simultaneously across the entire cluster by setting the system variable [`tidb_ttl_running_tasks`](/system-variables.md#tidb_ttl_running_tasks-new-in-v700). However, not all TTL jobs for all kinds of tables can be split into tasks. Please refer to the [Limitations](#limitations) section to understand which kinds of tables' TTL jobs cannot be split into tasks.
```

Comment:
I added a new description for the TTL task (distributed execution) and the limitation for it. PTAL @lcwangchao @ran-huang

Reply:
lgtm